packages/common/src/types/collection.ts

```diff
@@ -81,7 +77,7 @@ export type TIndexRequest = {
 export type TIndexListRequest = TCollectionRequest;
 
 export type TIndexCreateRequest = TIndexRequest & {
-  index: TCollectionIndex;
+  index: TCollectionIndex[];
```

Comment:
`TCollectionIndex` is already a record that consists of multiple indexes so I don't think this need to be an array
```ts
export type TCollectionIndex<T = Record<string, any>> = Partial<
  Record<keyof T, ESorting>
>;

let a: TCollectionIndex<{ name: string; age: number }> = {
  age: ESorting.Asc,
  name: ESorting.Desc,
};
```